src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -118,10 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="SetColorSchemeParentCommandName" xml:space="preserve">
-    <value>Select color scheme</value>
+    <value>Apply color scheme</value>

```

Comment:
nit: why is this one "**Apply** color scheme" when the very similar opacity one is "**Change** background opacity"

Reply:
Changed based on feedback in [this thread](https://github.com/microsoft/terminal/pull/17715#discussion_r1718569912).

So far we've done...
❌`Apply color scheme`: not aligned with `Change background opacity` (feedback from current thread)
❌`Change color scheme`: `Change` doesn't imply that you'll be selecting a scheme (feedback from [this thread](https://github.com/microsoft/terminal/pull/17715#discussion_r1718569912))
❌`Select color scheme`: doesn't really say what you're doing with the scheme (feedback from me)
❌`Select color scheme...`: `...` is redundant with `>` (feedback from [this comment](https://github.com/microsoft/terminal/issues/16846#issuecomment-2005007519))

That said, we've had `Select color scheme...` around for like 4 years. I'm going to change this back to `Select color scheme` (no `...`) to maintain muscle memory.

Leaving thread unresolved to maintain a space for discussion.

Reply:
Changed back to `Select color scheme...`